spec/imsc-hrm.html

```diff
@@ -237,9 +237,169 @@ <h2>Terms and Definitions</h2>
+        exceed this limit;</li>
+  
+        <li>implementer can design their presentation processor to maximize the
+        likelihood that it will be able to render correctly all <a>IMSC Document
+        Instances</a> that do not exceed this limit; and</li>
+          
+        <li>audiences can expect subtitle and caption presentation to match
+        the authorial intention.</li>
+      </ul>
+
+    </section>
+  
+    <section>
+      <h3>Why is the HRM needed to limit complexity?</h3>
+
+      <p>The HRM supplements the syntactic and structural constraints imposed in
+      [[IMSC]] by imposing constraints on the contents of the presentation.</p>
+  
+      <p>Because of the temporal and spatial variability of subtitles and
+        captions across types of content, territories and languages, it is not
+        possible to limit the complexity of an <a>IMSC Document Instance</a> using
+        only average values.</p>
+  
+      <p class="example">An average-based constraint of 840 characters per
+      minute could be met in multiple ways, with different rendering
+      complexities. Contrast two potential approaches. In the first, 5
+      characters are presented for a fraction of a second, followed by 835
+      characters that are then presented for over 59 seconds. This generates a
+      high rendering complexity for the 835 characters, since there is only a
+      brief time available to paint them. In the second, 210 characters must be
+      painted every 15 seconds, giving 15 seconds to prepare for the next
+      presentation. This has a much lower rendering complexity.</p>
+  
+      <p>The HRM achieves a more accurate representation of the complexity of an
+        <a>IMSC Document Instance</a> at any given time by taking into account
+        its past complexity in addition to its instantaneous complexity. The
+        same approach is commonly used in video to limit bitstream complexity,
+        e.g., the Hypothetical Reference Decoder (HRD) specified in
+        [[iso14496-10]].
+      </p>
+    </section>
+  
+    <section>
+      <h3>How does the HRM measure complexity?</h3>
+  
+      <p>
+        The HRM defines an hypothetical time required for painting subtitles and
+        captions. Painting includes drawing region backgrounds, rendering and
+        copying text, and decoding and copying images. Complexity is then
+        limited by requiring that the hypothetical time to paint a
+        subtitle/caption is shorter than the time elapsed since the previous
+        subtitle/caption.
+      </p>
+  
+      <p>The hypothetical time required for painting subtitles and captions is
+      based on a static analysis of the <a>IMSC Document Instance</a> and
+      requires no fetching of external resources or actual rendering. It assumes
+      that some characters are faster to render than others, based on whether
+      they were previously rendered and on the kind of glyph they generate.<p>
```

Comment:
Following merge of #42 we should revisit this paragraph. 

`character` and `glyph` are defined terms, so should be turned into `<a>` links. 

More importantly, since this is introductory text, I think it's likely to cause confusion about the model for understanding characters and glyphs. I think we need to say that it assumes a one to one mapping from character to glyph, and accounts for that assumption being, in general, false, by assigning different render speeds for different scripts.

Reply:
See improved text at https://github.com/w3c/imsc-hrm/commit/bdd49671c746f9ae3e99df5ef17952c96b0ce423